### DISCUSSION TITLE
Colocar um IP em Whitelist no Agent Wazuh

### DISCUSSION THREAD

**Discussion by grillado** [wazuh/wazuh#17276] [General]
Saudações!
Bom dia!
Vejam se podem me ajudar, eu preciso que os agentes do Wazuh ignorem o IP do Nessus que faz um scan legítimo no ambiente.
Não consegui fazer isso.
Desde já, muito grato.

**Comment by mdiego92** [comment]
Hello @grillado,

Which rules do you want to ignore the Nessus IP? Could you please show me an example so I can help you?

Looking forward to your response!

**Reply by grillado** [reply]
Olá, ,diego92!
Fico muito agradecido pelo apoio, normalmente o Nessus tenta as técnicas do Mitre. 
Defense Evasion, Persistence, Privilege Escalation, Initial Access, Brute Force...
Obrigado

**Reply by mdiego92** [reply]
Hello @grillado,

Could you please try adding the following rule to your `local_rules.xml` file or your custom rule file?

```
<group name="local">

  <rule id="100501" level="0">
    <match>192.168.1.1</match>
    <description>Ignore event from IP address.</description>
  </rule>

</group>
```

You will need to replace the **192.168.1.1** with your source IP address. This rule will mute all the events that have the modified IP address in the logs.

After that, please restart the Wazuh manager.

Looking forward to your response!

**Reply by grillado** [reply]
O meu arquivo é o local_rules.xml mesmo..
Adicionei aqui, vou realizar uns testes antes assim que voltar do almoço.
Muitíssimo obrigado, @mdiego92  !!!!

**Comment by grillado** [comment]
Resolvido!
@mdiego92 !!!!
